src/index.js

```diff
@@ -65,6 +66,30 @@ const removeEventListener = (event) => {
   }
 };
 
+const showInterstitial = (adUnitIdentifier, placement, customData) => {
```

Comment:
1. I think it is fine to use the shorthand `adUnitId`, we only spell out "identifier" in iOS to adhere to Apple coding conventions.
2. Are you sure we need all these conditionals? I can't remember the specifics but is it possible to have only one `showInterstitial(...)` function which takes in 3 arguments? Then pubs can do something like:

```
AppLovinMAX.showInterstitial(adUnitId, null, "MY_CUSTOM_DATA");
AppLovinMAX.showInterstitial(adUnitId, "MY_PLACEMENT", null);
AppLovinMAX.showInterstitial(adUnitId, "MY_PLACEMENT");
AppLovinMAX.showInterstitial(adUnitId);
```

If the parameter is not passed, it'll just evaluate to undefined.

Reply:
Fixed with the variable arguments.